en/controllers/components/security.rst

```diff
@@ -72,6 +72,12 @@ The ``$type`` parameter can have the following values:
   error.
 * 'secure' Indicates an SSL method restriction failure.
 
+.. versionadded:: cakephp/cakephp 2.3.6
```

Comment:
Thanks for the note, they were fixed here: https://github.com/cakephp/docs/commit/8f3c26958663b02066911164c4e51c34117c6dfd